web/src/elements/wizard/Wizard.ts

```diff
@@ -123,31 +184,72 @@ export class Wizard extends ModalButton {
         });
     }
 
+    /**
+     * Reset the wizard to it's initial state.
+     */
+    reset = () => {
+        this.open = false;
+
+        this.querySelectorAll("[data-wizardmanaged=true]").forEach((el) => {
+            el.remove();
+        });
+
+        for (const step of this.steps) {
+            const stepElement = this.getStepElementByName(step);
+
+            stepElement?.reset?.();
```

Comment:
I contemplated performing a `querySelectorAll('form')` but suspected that it would go against the Shadow DOM encapsulation. Given that each step element acts like a form element, it felt appropriate to give them a `reset` method.